app.js

```diff
@@ -8,15 +8,20 @@ app.set('view engine', 'pug')
 app.use(express.urlencoded())
 app.use(cookieSession({ secret: 'superdupersecret' }));
 
+const server = app.listen(port, () => {
+  console.log(`Chat app is listening on port ${port}`)
+})
+
+const io = require('socket.io')(server);
```

Comment:
It's helpful to have all of your imports in the same place so you can see what the code requires to run at a glance. Typically at the top of the file.

Reply:
Will address in next PR.